src/cbmc/cbmc_parse_options.h

```diff
@@ -48,6 +48,7 @@ class optionst;
   "(document-subgoals)(outfile):(test-preprocessor)" \
   "D:I:(c89)(c99)(c11)(cpp98)(cpp03)(cpp11)" \
   "(object-bits):" \
+  "(malloc-may-fail)" \
```

Comment:
:grey_question: should this option be available in other C based tools, like goto-analyzer, or does that not really make sense?

Reply:
Ambivalent.